modules/ossm-security-cert-manage.adoc

```diff
@@ -44,97 +45,94 @@ spec:
+$ oc -n istio-system delete pods -l 'app in (istiod,istio-ingressgateway, istio-egressgateway)
 ----
 +
-. Run the following commands to retrieve the certificates mounted on the proxy.
-+
-[source,terminal]
-----
-$ oc exec -it $RATINGSPOD -c istio-proxy -- /bin/cat /var/run/secrets/istio/root-cert.pem > /tmp/pod-root-cert.pem
-----
-+
-The file `/tmp/pod-root-cert.pem` contains the root certificate propagated to the pod.
+The Operator will automatically recreate the pods after they have been deleted.
+
+. Restart the bookinfo application pods so that the sidecar proxies pick up the secret changes. Use the following command to restart the pods:
 +
 [source,terminal]
 ----
-$ oc exec -it $RATINGSPOD -c istio-proxy -- /bin/cat /etc/certs/cert-chain.pem > /tmp/pod-cert-chain.pem
+$ oc -n bookinfo delete pods --all
 ----
 +
-The file `/tmp/pod-cert-chain.pem` contains the workload certificate and the CA certificate propagated to the pod.
-+
-. Verify the root certificate is the same as the one specified by the Operator. Replace `<path>` with the path to your certificates.
+You should see output similar to the following:
 +
+
 [source,terminal]
 ----
-$ openssl x509 -in <path>/root-cert.pem -text -noout > /tmp/root-cert.crt.txt
+pod "details-v1-6cd699df8c-j54nh" deleted
+pod "productpage-v1-5ddcb4b84f-mtmf2" deleted
+pod "ratings-v1-bdbcc68bc-kmng4" deleted
+pod "reviews-v1-754ddd7b6f-lqhsv" deleted
+pod "reviews-v2-675679877f-q67r2" deleted
+pod "reviews-v3-79d7549c7-c2gjs" deleted
 ----
+
+. Verify that the pods were created and are ready with the following command:
 +
+
 [source,terminal]
 ----
-$ openssl x509 -in /tmp/pod-root-cert.pem -text -noout > /tmp/pod-root-cert.crt.txt
+$ oc get pods -n bookinfo
 ----
+
+[id="ossm-cert-manage-verify-cert_{context}"]
+== Verifying your certificates
+
+Use the Bookinfo sample application to verify that the workload certificates are signed by the certificates that were plugged into the CA. This requires you have `openssl` installed on your machine
+
+. To extract certificates from bookinfo workloads use the following command:
 +
 [source,terminal]
 ----
-$ diff /tmp/root-cert.crt.txt /tmp/pod-root-cert.crt.txt
+$ sleep 60
+$ oc -n <bookinfo-project> exec "$(oc -n <bookinfo-project> get pod -l app=productpage -o jsonpath={.items..metadata.name})" -c istio-proxy -- openssl s_client -showcerts -connect details:9080 > bookinfo-proxy-cert.txt
+$ sed -n '/-----BEGIN CERTIFICATE-----/{:start /-----END CERTIFICATE-----/!{N;b start};/.*/p}' bookinfo-proxy-cert.txt > certs.pem
+$ awk 'BEGIN {counter=0;} /BEGIN CERT/{counter++} { print > "proxy-cert-" counter ".pem"}' < certs.pem
 ----
 +
-Expect the output to be empty.
-+
-. Verify the CA certificate is the same as the one specified by Operator. Replace `<path>` with the path to your certificates.
+After running the command, you should have three files in your working directory: `proxy-cert-1.pem`, `proxy-cert-2.pem` and `proxy-cert-3.pem`.
```

Comment:
"After running the command, you should have three files in your working directory: proxy-cert-1.pem, proxy-cert-2.pem and proxy-cert-3.pem."

I have bookinfo-proxy-cert.txt, certs,pem, proxy-cert-1.pem, proxy-cert-2.pem, so I'm not sure what's going on here.  Again, it's probably time to loop QE back in so we can get their help figuring what's going on here.